frontend/src/src/components/Map.svelte

```diff
@@ -1,140 +1,147 @@
-
-
-onMount(async () => {
-  console.log('init map');
-  // To add the draw toolbar set the option drawControl: true in the map options.
-  map = L.map('map', {zoomControl: false});
-  map.setView(INITIAL_MAP_CENTER, INITIAL_ZOOM);
-  map.addLayer(baseLayersGroup);
-  map.addLayer(selectionsGroup);
-  map.addLayer(overlaysGroup);
-  const baseLayer = L.tileLayer(BASE_LAYER_URL, BASE_LAYER_PARAMS);
-  baseLayersGroup.addLayer(baseLayer); // Add the openstreetmap layer
-  // Add the map controls
-  map.addControl(makeSearchControl()); // Search tools
-  map.addControl(makeCMToggleControl()); // Button to open calculation module pane
-  map.addControl(makeAreaSelectionControl());
-  map.addControl(makeOverlayLayersControl());
-});
-function resizeMap() {
-  if (map) {
-    map.invalidateSize();
+  import {onMount} from 'svelte';
+  // Import CSS from Leaflet and plugins.
+  import 'leaflet/dist/leaflet.css';
+  import 'leaflet.markercluster/dist/MarkerCluster.css';
+  import 'leaflet.markercluster/dist/MarkerCluster.Default.css';
+
+  // Import JS from Leaflet and plugins.
+  import 'leaflet/dist/leaflet';
+  import 'leaflet.markercluster/dist/leaflet.markercluster';
+  import 'leaflet.gridlayer.googlemutant/Leaflet.GoogleMutant';
+  import 'leaflet-search/dist/leaflet-search.src.js';
+  import 'leaflet-search/dist/leaflet-search.src.css';
+
+  import AreaSelection from './AreaSelection.svelte';
+  import LayerSelection from './LayerSelection.svelte';
+  import CMToggle from './CMToggle.svelte';
+  import TopNav from './TopNav.svelte';
+  import {activeOverlayLayersStore, activeSelectionLayerStore} from '../stores.js';
+
+  import {INITIAL_MAP_CENTER, INITIAL_ZOOM, BASE_LAYER_URL} from '../settings.js';
+  import {BASE_LAYER_PARAMS} from '../settings.js';
+
+  let map;
+  $: activeSelectionLayer = $activeSelectionLayerStore;
+  $: activeOverlayLayers = $activeOverlayLayersStore;
+
+  const overlaysGroup = L.layerGroup();
+  const selectionsGroup = L.layerGroup();
+  const baseLayersGroup = L.layerGroup();
+
+
+  onMount(async () => {
+    console.log('init map');
+    // To add the draw toolbar set the option drawControl: true in the map options.
+    map = L.map('map', {zoomControl: false});
+    map.setView(INITIAL_MAP_CENTER, INITIAL_ZOOM);
+    map.addLayer(baseLayersGroup);
+    map.addLayer(selectionsGroup);
+    map.addLayer(overlaysGroup);
+    const baseLayer = L.tileLayer(BASE_LAYER_URL, BASE_LAYER_PARAMS);
+    baseLayersGroup.addLayer(baseLayer); // Add the openstreetmap layer
+    // Add the map controls
+    map.addControl(makeSearchControl()); // Search tools
+    map.addControl(makeCMToggleControl()); // Button to open calculation module pane
+    map.addControl(makeAreaSelectionControl());
+    map.addControl(makeOverlayLayersControl());
+  });
+
+  function resizeMap() {
+    if (map) {
+      map.invalidateSize();
+    }
+  }
+
+  $: {
+    console.log(`selected layer was changed: ${activeSelectionLayer}`);
```

Comment:
On another PR, I would suggest to remove all those `console.log()` everywhere from the code.

Reply:
Either that or fix them, since most print something like: `[object Object]`